content/member/susanto.md

```diff
@@ -0,0 +1,61 @@
++++
+bio = ""
+date = "2023-10-11"
+id = "susanto"
+interests = ["XAI", "Autonomous Vehicles", "NLP"]
+name = "Andrew Susanto"
+portrait = "/portraits/leilani.jpg"
```

Comment:
I think you need to change this to andrew :P

Reply:
Hahaha I will do that! Thank you! Sorry I missed it : P